internal/contour/translator.go

```diff
@@ -208,6 +215,14 @@ func (t *Translator) removeSecret(s *v1.Secret) {
 	t.recomputeTLSListener(t.cache.ingresses, t.cache.secrets)
 }
 
+func (t *Translator) addRoute(s *v1alpha1.Route) {
+	fmt.Printf("-------- ADD ROUTE: %s\n", s.ObjectMeta.Name)
```

Comment:
t.Infof // now we have logrus plumbed through

Reply:
I can update, was trying to be explicit that it was stubbed out.